README.md

```diff
@@ -1,15 +1,22 @@
-Android Key Attestation Sample
+Android Key Attestation Library
 ===================================
 
-This sample illustrates how to use the [Bouncy Castle ASN.1][1] parser to extract information
+This library uses the [Bouncy Castle ASN.1][1] parser to extract information
 from an Android attestation data structure to verify that a key pair has been
-generated in an Android device. 
+generated in an Android device. It is maintained in tandem with Android's key
+attestation capabilities and is meant for production use.
 
-This repository contains a [server](server/) sample that shows how to attest an Android certificate
-outside the Android framework. This is the recommended best practice, since if the Android device
-is rooted or otherwise compromised on-device validation of the attestation may be inaccurate.
+This repository contains a [server](server/src/main/java/com/android/example/)
+sample code that shows how to validate an Android attestation certificate chain
+outside the Android framework. This is the recommended best practice, since if
+the Android device is rooted or otherwise compromised on-device validation of
```

Comment:
nit: missing comma between "compromised" and "on-device".

Reply:
Done.

Reply:
Done.